bin/pre-pull-crunchy-containers.sh

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-for CNAME in crunchy-postgres crunchy-collect crunchy-pgbadger crunchy-pgbouncer crunchy-grafana crunchy-prometheus
+for CNAME in crunchy-postgres crunchy-pgbadger crunchy-pgbouncer crunchy-grafana crunchy-prometheus
```

Comment:
I believe we can take out `crunchy-grafana` and `crunchy-prometheus` here as well, though in a separate commit.

Reply:
Sure thing, I added that as a separate commit.